news/13653-check-content-length

```diff
@@ -0,0 +1,20 @@
+### Enhancements
+
+* <news item>
+
+### Bug fixes
+
+* Check `Content-Length` is nonzero before calculating progress,
+  avoiding a possible `ZeroDivisionError`. (#13653, #13671)
```

Comment:
Oh, could you please just update the CHANGELOG directly?

Reply:
Yes